### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by ahdinosaur** [nodeschool/wellington#1]
we need a sweet logo to use for branding that can fit on [hexagon stickers](https://terinjokes.github.io/StickerConstructorSpec/#type-hexagon)!

one option is we fork [the NodeSchool template sticker](https://github.com/nodeschool/nodeschool.github.io/tree/source/images/make-a-sticker).

i have another idea: we use the Wellington bucket fountain design from the [What is Enspiral?](https://vimeo.com/125088390) video, then add "NodeSchool" in text somewhere. to this end, i've gotten permission and files from [Rachelle Smith](https://www.linkedin.com/pub/rachelle-smith/75/a2a/574) for the bucket fountain design: [svg](https://github.com/nodeschool/wellington/blob/master/media/bucket-fountain.svg), [png](https://github.com/nodeschool/wellington/blob/master/media/bucket-fountain.png).

![bucket fountain design](https://raw.githubusercontent.com/nodeschool/wellington/master/media/bucket-fountain.png)

any other ideas?

**Comment by ahdinosaur** [comment]
played around a little bit: [svg](https://github.com/nodeschool/wellington/blob/master/media/logo-pre0.svg)

![preliminary logo 0](https://rawgithub.com/nodeschool/wellington/master/media/logo-pre0.svg)

used parts of template, [NodeSchool toronto logo](https://github.com/nodeschool/nodeschool.github.io/blob/source/images/nodeschool-toronto.svg) and bucket fountain.

i reckon needs some solid-line waves, bubbles and/or winds to fill in the empty space, hmm.

**Comment by ahdinosaur** [comment]
continued along the above train of thinking, also fixed up the WLG text:

[![preliminary logo 1](https://rawgithub.com/nodeschool/wellington/master/media/logo-pre1.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre1.svg)

used [wind](https://thenounproject.com/search/?q=wind&i=7702) from [NounProject](https://thenounproject.com/).

**Comment by mmckegg** [comment]
:+1: nicee!

**Comment by ahdinosaur** [comment]
idea from @ashlynbaum: fern leaf instead of bubbles. findings: [1](http://commons.wikimedia.org/wiki/File:Silver_Fern_flag.svg) [2](https://en.wikipedia.org/wiki/File:Koru_flag.svg)

**Comment by ahdinosaur** [comment]
another iteration:

[![preliminary logo 2](https://rawgit.com/nodeschool/wellington/master/media/logo-pre2.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre2.svg)

used [Koru flag](https://en.wikipedia.org/wiki/File:Koru_flag.svg).

**Comment by ahdinosaur** [comment]
more iterations:

[![preliminary logo 3](https://rawgit.com/nodeschool/wellington/master/media/logo-pre3.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre3.svg)

[![preliminary logo 4](https://rawgit.com/nodeschool/wellington/master/media/logo-pre4.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre4.svg)

**Comment by mmckegg** [comment]
can you try that last one with wind in the sky instead?

And maybe try changing the WLG to white

Possibly the colour of the sea should be the same as the NODESCHOOL text.

**Comment by ashlynbaum** [comment]
Here is another take on the the wind idea.

[![nodeschool_wellington](https://rawgit.com/nodeschool/wellington/master/media/logo-pre5.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre5.svg)

**Comment by mmckegg** [comment]
@ashlynbaum: nice I and simple! I like it!

**Comment by max-mapper** [comment]
my personal fave is ![preliminary logo 2](https://rawgit.com/nodeschool/wellington/master/media/logo-pre2.svg), i feel it has the most NZ vibes

**Comment by mmckegg** [comment]
I really like the big bucket fountain in @ashlynbaum's design. @ahdinosaur why not try [this](https://camo.githubusercontent.com/af7acaaa70857a01ff9190830d9a50e7eb8e0378/68747470733a2f2f7261776769742e636f6d2f6e6f64657363686f6f6c2f77656c6c696e67746f6e2f6d61737465722f6d656469612f6c6f676f2d707265322e737667) with a larger bucket fountain (removing the lines at the top)

**Comment by ahdinosaur** [comment]
@mmckegg:

[![preliminary logo 6](https://rawgit.com/nodeschool/wellington/master/media/logo-pre6.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre6.svg)

**Comment by ahdinosaur** [comment]
@mmckegg with your other suggestions:

[![preliminary logo 7](https://rawgit.com/nodeschool/wellington/master/media/logo-pre7.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre7.svg)

**Comment by mmckegg** [comment]
@ahdinosaur think I like pre6 best :smile: 

pre7: The water just seems a bit too intense using the same colour as above (now that I see it).

**Comment by ashlynbaum** [comment]
@ahdinosaur, I agree with @mmckegg  its too hard to read the WLG  with the blue bottom. And it feels a little cluttered.  I think If there is a blue bottom, the buckets should be smaller like in the version 2.

**Comment by ahdinosaur** [comment]
more background experimentation:

[![preliminary logo 8](https://rawgit.com/nodeschool/wellington/master/media/logo-pre8.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre8.svg)

**Comment by ahdinosaur** [comment]
@genevievebelle thinks the koru isn't needed, wonders about having just wind on both sides.

**Comment by ahdinosaur** [comment]
@genevievebelle

[![preliminary logo 9](https://rawgit.com/nodeschool/wellington/master/media/logo-pre9.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre9.svg)

**Comment by genevievebelle** [comment]
Yeah I like that! What about with a coloured background?

**Comment by mixmix** [comment]
anudd option

(note the accurate thinkpad / macbook color backgrounds)

![silver_bucket](https://cloud.githubusercontent.com/assets/2665886/8200411/b91973d2-1518-11e5-97b7-3d02e0de8eb5.jpg)
![black-bucket](https://cloud.githubusercontent.com/assets/2665886/8200412/b919c170-1518-11e5-8de8-e3edc212fea5.jpg)

**Comment by genevievebelle** [comment]
Nice! Liking the 3D box effect @mixmix , are your inkscape skillz up to that @ahdinosaur ??

**Comment by mixmix** [comment]
channeling @pfraze 's sweet avatar :  
![selection_261](https://cloud.githubusercontent.com/assets/2665886/8200470/811508b0-1519-11e5-884d-e56707e7c816.png)

**Comment by mixmix** [comment]
we could add water to the 3d cube .. like imaging a fish tank with water partly filling it 
like 

![tank_water](https://cloud.githubusercontent.com/assets/2665886/8200854/c8ca79ce-151e-11e5-9975-f0fe44f8e573.JPG)

**Comment by mmckegg** [comment]
@mixmix: I like where this is going!

**Comment by mixmix** [comment]
how about some clouds and wellington in there too ?
![wellington_cube](https://cloud.githubusercontent.com/assets/2665886/8201039/c93ff40e-1520-11e5-9197-de62a580ca06.JPG)
![wellington_cube_black](https://cloud.githubusercontent.com/assets/2665886/8201076/39db5ff0-1521-11e5-95bb-c93e71af4717.JPG)

**Comment by ahdinosaur** [comment]
@mixmix this is incredible, so keen. :smile_cat:

**Comment by pfrazee** [comment]
that's awesome

**Comment by ahdinosaur** [comment]
my flatmate, Clemence, contributed another preliminary logo :smile: 

[![preliminary logo 10](https://rawgit.com/nodeschool/wellington/master/media/logo-pre10.svg)](https://github.com/nodeschool/wellington/blob/master/media/logo-pre10.svg)

**Comment by ashlynbaum** [comment]
So what was the final decision on the logo?

**Comment by ahdinosaur** [comment]
no decision yet, i haven't had time to attempt @mixmix's design in inkscape, also leaving some breathing room for any more logos ideas.

**Comment by zvyn** [comment]
another proposal with less colors and a stronger reference to the parent organisation:
![g5935](https://cloud.githubusercontent.com/assets/1493900/8691595/0d3fadb2-2b19-11e5-9a16-13addc8865dc.png)

svg is here: http://pastebin.com/raw.php?i=JxvHkPJh
